net/core/dev.c

```diff
@@ -9857,6 +9963,23 @@ static int __init net_dev_init(void)
 		init_gro_hash(&sd->backlog);
 		sd->backlog.poll = process_backlog;
 		sd->backlog.weight = weight_p;
+
+		new_state_cpu = (lua_state_cpu *) kmalloc(sizeof(lua_state_cpu),
+							  GFP_KERNEL);
```

Comment:
I'm not sure if `GFP_KERNEL` can be used in this context

Reply:
You think it needs to be `GFP_ATOMIC`?